api/src/utils/file-utils.ts

```diff
@@ -272,15 +272,15 @@ export async function getObjectMeta(key: string): Promise<HeadObjectOutput> {
  * @param {IArtifactS3FileKey} options
  * @return {*}
  */
-export function generateArtifactS3FileKey(options: IArtifactS3FileKey) {
+export function generateSubmissionFeatureS3FileKey(options: IArtifactS3FileKey) {
   const keyParts: (string | number)[] = [];
 
   keyParts.push(_getS3KeyPrefix());
-  keyParts.push('datasets');
-  keyParts.push(options.datasetUUID);
-  keyParts.push('artifacts');
+  keyParts.push('submissions');
```

Comment:
Can we simplify this function a bit:

```javascript
return [
  'datasets',
  options.datasetUUID,
  // ...
].filter(Boolean).join('/');
```

Reply:
For sure, the main reason for it being separate pushes, historically, was that we had some if statements in there that would change the key depending on which fields were provided. Something we borrowed from SIMS. Though in BioHub it looks like we never had any optional fields, so there were no conditional key parts.